lib/Feature/XmlParser.cpp

```diff
@@ -30,13 +31,37 @@ void XmlParser::parseConfigurationOption(xmlNode *N, bool Num = false) {
           if (Child->type == XML_ELEMENT_NODE) {
             if (!xmlStrcmp(Child->name,
                            reinterpret_cast<constXmlCharPtr>("options"))) {
-              std::unique_ptr<xmlChar, void (*)(void *)> CContent(
+              std::unique_ptr<xmlChar, void (*)(void *)> CCnt(
                   xmlNodeGetContent(Child), xmlFree);
-              RawExcludes.emplace_back(
-                  Name, reinterpret_cast<char *>(CContent.get()));
+              RawExcludes.emplace_back(Name,
+                                       reinterpret_cast<char *>(CCnt.get()));
             }
           }
         }
+      } else if (!xmlStrcmp(Head->name,
+                            reinterpret_cast<constXmlCharPtr>("location"))) {
+        std::string Path;
+        std::optional<std::pair<int, int>> Start = std::nullopt;
+        std::optional<std::pair<int, int>> End = std::nullopt;
```

Comment:
```suggestion
        std::optional<std::pair<int, int>> Start;
        std::optional<std::pair<int, int>> End;
```
Default construction already gives you an empty optional.